command/helper/config.go

```diff
@@ -56,6 +57,11 @@ type TxPool struct {
 	MaxSlots   uint64 `json:"max_slots"`
 }
 
+// Headers defines the HTTP response headers required to enable CORS.
+type Headers struct {
+	AccessControlAllowOrigin []string `json:"access_control_allow_origin"`
```

Comment:
`AccessControlAllowOrigins` would be more suitable?

Reply:
Nice catch :100:, fixed in https://github.com/0xPolygon/polygon-edge/pull/368/commits/381d5af44c6de770fdebf50a1d7e90b2af538b53